etc/adminhtml/system.xml

```diff
@@ -110,9 +110,30 @@
                             <field id="active">1</field>
                         </depends>
                     </field>
-                    <field id="pricing_strategy" showInDefault="1" showInStore="1" showInWebsite="1" sortOrder="100" translate="label" type="select">
```

Comment:
does this mean that this setting will only be shown in the default config and not the store/website config? I'm not sure we want to restrict it to only the default view but I wanted to confirm first before checking with DHL